public/app/features/dashboard/state/PanelModel.ts

```diff
@@ -232,16 +232,20 @@ export class PanelModel implements DataConfigSource {
     return this.fieldConfig;
   }
 
+  get hasChanged(): boolean {
```

Comment:
@kaydelaney -- I changed the bool to a counter so we can quickly check if all configs are the same and then added this helper so your existing checks look reasonable